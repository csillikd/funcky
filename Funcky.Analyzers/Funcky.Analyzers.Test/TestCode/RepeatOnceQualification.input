namespace Funcky
{
    class Sequence
    {
        public static System.Collections.Generic.IEnumerable<T> Return<T>(T value) => null!;
    }
}

namespace WithUsingStatic
{
    using static Funcky.Sequence;

    public class C
    {
        public void M()
        {
            _ = System.Linq.Enumerable.Repeat(10, 1);
        }
    }
}

namespace WithFullyQualifiedName
{
    public class C
    {
        public void M()
        {
            _ = System.Linq.Enumerable.Repeat(10, 1);
        }
    }
}

namespace WithUsingAlias
{
    using Seq = Funcky.Sequence;

    public class C
    {
        public void M()
        {
            _ = System.Linq.Enumerable.Repeat(10, 1);
        }
    }
}
