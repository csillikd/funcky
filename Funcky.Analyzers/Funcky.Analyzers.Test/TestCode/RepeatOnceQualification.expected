namespace Funcky
{
    class Sequence
    {
        public static System.Collections.Generic.IEnumerable<T> Return<T>(T value) => null!;
    }
}

namespace WithUsingStatic
{
    using static Funcky.Sequence;

    public class C
    {
        public void M()
        {
            _ = Return(10);
        }
    }
}

namespace WithFullyQualifiedName
{
    public class C
    {
        public void M()
        {
            _ = Funcky.Sequence.Return(10);
        }
    }
}

namespace WithUsingAlias
{
    using Seq = Funcky.Sequence;

    public class C
    {
        public void M()
        {
            _ = Seq.Return(10);
        }
    }
}
